app/modules/intelligence/tools/local_search_tools/search_semantic_tool.py

```
@@ -0,0 +1,115 @@
+            name = r.get("name", "")
+            
+            formatted += f"{i}. **{file_path}:{start_line}**"
+            if name:
+                formatted += f" - `{name}`"
+            formatted += f" (similarity: {similarity:.3f})\n"
+            
+            if docstring:
+                formatted += f"   {docstring[:200]}{'...' if len(docstring) > 200 else ''}\n"
+            formatted += "\n"
+        
+        return formatted
```

Ruff (0.15.0)

[warning] Consider moving this statement to an `else` block

(TRY300)